### PR TITLE
SILA-4289: Implement Generate a Processor Token, Already have a plaid account flow.

### PR DESCRIPTION
**Pull request by ArcGate-Vinod**
Hi @amack300,

Implement `Generate a Processor Token`, `Already have a plaid account` flow.

Noted: start `node` backend server to test out this PR.

Thanks!

### CODE REVIEW

File: src/components/accounts/GenerateProcessorComponents.js [hunk 1]
```diff
@@ -397,10 +399,8 @@ export const LinkProcessorToken = ({ step, title, context, allPlaidTokens, onHan
   </>);
 };
 
-export const plaidSignUpSteps = [{
-    title: 'Plaid + Sila Account',
-    disabled: false,
-    context: [{
+const COMMON_CONTENT_AND_LINKS = [{
```
Comment (amack300) [inline src/components/accounts/GenerateProcessorComponents.js:402]: Hey @ArcGate-Vinod, can we move this into its own file in constants/plaidSignUpSteps.js and import it.

Comment (ArcGate-Vinod) [reply src/components/accounts/GenerateProcessorComponents.js:402]: Hi @amack300,
This change has been addressed.
Thanks!

File: src/views/Accounts.js [hunk 2]
```diff
@@ -334,10 +336,19 @@ const Accounts = ({ page, previous, next, isActive }) => {
...
+      setGenerateProcessorPages({ ...generateProcessorPages, gpRoutes: havePlaidAccountSteps.map((r, index) => (index === 0 || index === 1) ? { ...r, disabled: false } : { ...r, disabled: true }) });
+      setTabKey(1);
+    } else if (!key && type && type === 'accounts') {
+      setGenerateProcessorPages({ ...generateProcessorPages, isGpPage: false, gpRoutes: DEFAULT_GP_ROUTES });
+      setTabKey(0);
+    } else if (!key && type && type === 'goBack') {
+      setGenerateProcessorPages({ ...generateProcessorPages, gpRoutes: DEFAULT_GP_ROUTES });
+      setTabKey(0);
     } else if (key) {
-      generateProcessorPages.gpRoutes[key]['disabled'] = false;
```
[the remark this answers is not in the document]
Comment (ArcGate-Vinod) [reply src/views/Accounts.js:335]: Hi @amack300,
This change has been addressed.
Thanks!

**Review by amack300 — APPROVED** [review]
LGTM, thanks for your changes.

File: src/views/Accounts.js [hunk 3]
```diff
@@ -334,10 +336,19 @@ const Accounts = ({ page, previous, next, isActive }) => {
...
       setTabKey(key);
+      generateProcessorPages.gpRoutes[key]['disabled'] = false;
```
[the remark this answers is not in the document]
Comment (ArcGate-Vinod) [reply src/views/Accounts.js]: Hi @amack300,
This change has been addressed.
Thanks!

File: src/views/Accounts.js [hunk 4]
```diff
@@ -334,10 +336,19 @@ const Accounts = ({ page, previous, next, isActive }) => {
 
   const handleClick = (type, key) => {
```
[the remark this answers is not in the document]
Comment (ArcGate-Vinod) [reply src/views/Accounts.js:333]: Hi @amack300,

This above changes have addressed in this PR - https://github.com/Sila-Money/sila-sdk-demo/pull/181

Thanks!

**Pull request merged**